.coderabbit.yml

```
@@ -0,0 +1,8 @@
+ignore:
+  - "**/*.stories.*"
+  - "**/*.test.*"
+  - "**/*.spec.*"
+  - "**/__tests__/**"
+  - "**/tests/**"
+  - "**/__stories__/**"
+  - "**/stories/**"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)